src/tools/generate/remove_unnecessary_markup.js

```diff
@@ -0,0 +1,10 @@
+/* 
+  TODO: This is a workaround for the unnecessary markup that's being generated 
+  by JSDOM. Once that has been fixed, this can hopefully be removed. 
+  (See #412) 
+*/
+const remove_unnecessary_markup = (html) => html.replace(/<p><\/p>/g, '');
```

Comment:
Should we make this more generic under the assumption there might be other hacks to come and explain each hack with a comment? Or is this overkill and can be done on the next hack (if there is one)?

```suggestion
const remove_unnecessary_markup = (html) => {
  // Remove pointless paragraph tags added by JSON in wrap_tables
  html = html.replace(/<p><\/p>/g, '');
  return html
}
```

Reply:
We definitely should. :P